static/css/style.css

```diff
@@ -116,6 +123,25 @@ a:hover {
 
 /* Entry Review Page */
 
+.entry-container {
+    margin-top: 2rem;
+}
+
+.delete-entry {
+    text-transform: uppercase;
+}
+
+.entry .delete-btn {
```

Comment:
I appreciate this was not issue in this pull-request, but do we need to scope the `.delete-btn` selector with `.entry`? Do we have `.delete-btn` elements which are not within a `.entry` and need to be styled differently?